src/components/InputPassword.js

```diff
@@ -126,23 +126,15 @@ class InputPassword extends Component {
               text="lower"
               open={this.state.showRequirements}
             >
-              <span
-                valid={
-                  this.state.password.toUpperCase() !== this.state.password
-                }
-              />
+              <span valid={this.state.password.toUpperCase() !== this.state.password} />
```

Comment:
going to check, this is one of those Prettier formatting changes, maybe its actually good ahah makes use double code review